packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md

```diff
@@ -0,0 +1,7 @@
+To use the Okta CLI:
+1. Enter the command: `okta apps create`
+2. Give your app a name.
+3. Select **Web** for Type of Application.
+4. Select **Other**.
+5. Choose the default Login redirect URI, `http://localhost:8080/callback`
```

Comment:
```suggestion
5. Choose the default Login redirect URI, `http://localhost:8080/login/callback`
```
@aarongranick-okta and @swiftone agreed last week that `/login/callback` is a better default.